specification/purview/data-plane/Azure.Analytics.Purview.DevopsPolicies/preview/2022-11-01-preview/purviewDevopsPolicy.json

```
@@ -0,0 +1,611 @@
+          }
+        },
+        "x-ms-examples": {
+          "DevopsSqlSecurityAuditorPolicy_CreateOrReplace": {
+            "$ref": "./examples/DevopsSqlSecurityAuditorPolicy_CreateOrReplace.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DevopsPolicy"
+        ],
+        "description": "Deletes a devops sql security auditor policy",
+        "operationId": "DevopsSqlSecurityAuditorPolicy_Delete",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "purviewResourceName",
+            "description": "Name of resource as registered on the Azure purview portal",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DevopsSqlSecurityAuditorPolicy_Delete": {
+            "$ref": "./examples/DevopsSqlSecurityAuditorPolicy_Delete.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DevopsPolicy"
+        ],
+        "description": "Get devops sql security auditor policy",
+        "operationId": "DevopsSqlSecurityAuditorPolicy_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "purviewResourceName",
+            "description": "Name of resource as registered on the Azure purview portal",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DevopsPolicy"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DevopsSqlSecurityAuditorPolicy_Get": {
+            "$ref": "./examples/DevopsSqlSecurityAuditorPolicy_Get.json"
+          }
+        }
+      }
+    },
+    "/devopsPolicies/sqlPerfMonitorRoles/{purviewResourceName}": {
```

As above, I recommend
```suggestion
    "/devopsPolicies/{purviewResourceName}/sqlPerfMonitorRoles": {
```

---

Updated.